Doc/requirements.txt

```diff
@@ -7,6 +7,8 @@ sphinx==4.2.0
 
 blurb
 
+sphinx-lint
```

Comment:
Shouldn't this dependency be pinned? I say this because if we refactor sphinx-lint (e.g. as a Sphinx builder as discussed in https://github.com/sphinx-contrib/sphinx-lint/issues/1), it will require adaptation here. A new release will have to be made with the changes before they can be used in CPython, and between the release and the actual switch, things will be broken (plus people might not understand what's going on it if they don't pull recent changes but install a newer sphinx-lint).

Reply:
Another thought: it will make it easier to add checks that might have a few false positives (preferably there won't be any, but it could be tricky to eliminate some rare cases).

Reply:
> it will require adaptation here.

I don't think so, but I may miss something. sphinx-lint should be able to find the conf.py by itself, and we'll be able to add a sphinx install dependency, so it looks like it should be doable without adaptation.

Still we can pin `<1` and use semver, so if we break something we can bump the major version if you prefer.

Reply:
The normal way to invoke a Sphinx builder would basically be `sphinx-build -b lint`. I haven't investigated if it's feasible to allow direct invocation by a script; there would be implications on which Sphinx to use for example (from some venv or not?). I think it can't hurt to require `<1`.

Reply:
OK for `< 1`, done.